packages/manager/apps/pci-private-registry/src/pages/api-url/APIUrl.page.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  ODS_THEME_COLOR_INTENT,
+  ODS_THEME_TYPOGRAPHY_SIZE,
+} from '@ovhcloud/ods-common-theming';
+import {
+  ODS_BUTTON_VARIANT,
+  ODS_TEXT_COLOR_INTENT,
+  ODS_TEXT_LEVEL,
+} from '@ovhcloud/ods-components';
+import {
+  OsdsButton,
+  OsdsClipboard,
+  OsdsModal,
+  OsdsText,
+} from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate, useParams } from 'react-router-dom';
+import { useGetAllRegistries } from '@/api/hooks/useRegistry';
+
+export default function ApiUrlPage() {
+  const { t } = useTranslation();
```

Comment:
does useTranslation without a param defaults to common?

Reply:
yeah sure